app/i18n/locales/es/about-page.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "Acerca de nuestro equipo",
+  "description": "PSE es un laboratorio de investigación y desarrollo multidisciplinario apoyado por la Fundación Ethereum. Creamos infraestructura de código abierto, herramientas y recursos educativos para incorporar criptografía en aplicaciones del mundo real.",
+  "our-principles-title": "Nuestros principios",
+  "principles": [
+    {
+      "title": "01. Criptografía para personas",
+      "description": [
+        "La criptografía está en todas partes: cada vez que te conectas a un sitio seguro, inicias sesión con una contraseña o desbloqueas tu teléfono, estás viendo la criptografía en acción.",
+        "Con la criptografía 'programable' (como pruebas de conocimiento cero, computación multiparte o cifrado homomórfico), podemos hacer afirmaciones verificables sobre información secreta sin revelar la información en sí misma. Esto se puede aplicar a la gestión de identidad, resistencia a la colusión, comunicación anónima y mucho más.",
+        "Estamos construyendo una biblioteca de herramientas de desarrollo, artículos de investigación y prototipos que son de código abierto y gratuitos para que todos los usen. Esperamos que nuestros recursos inspiren a las personas a innovar en las tecnologías que sus comunidades necesitan."
+      ]
+    },
+    {
+      "title": "02. Privacidad por defecto",
+      "description": [
+        "Creemos que la privacidad es un derecho fundamental. Queremos ser parte de la construcción de un internet que se aleje de prácticas invasivas de datos y, en cambio, brinde a las personas opciones reales sobre quién tiene acceso a su información personal. La autorización debe ser específica para el propósito, revocable, informada y no coercitiva.",
+        "Creamos herramientas que ayudan a las personas a autenticarse de manera segura, realizar transacciones confidenciales en el blockchain y respetar y preservar la privacidad del usuario."
```

Comment:
_blockchain_ en cursiva si usamos el angliciscmo, sino cadena de bloques (suena mejor blockchain ;) )
https://www.rae.es/observatorio-de-palabras/blockchain

Reply:
Not sure if we can add cursive in this json, I'll leave it as cadena de bloques 🖖